go/protocol/keybase1/extras.go

```diff
@@ -3683,6 +3683,16 @@ func (s TeamSigChainState) GetAllUVs() (res []UserVersion) {
 	return res
 }
 
+func (s TeamSigChainState) ActiveInvites() (ret []TeamInvite) {
```

Comment:
this seems similar to `func (t *TeamSigChainState) ActiveInvites()` and seems to only be used in tests. but it's probably fine to keep both

Reply:
Up to you. I was about to make this return TeamInviteMetadata and have the chain.go one call this and then I got sad when I looked at the number of tests I had to fix haha

Reply:
But I can do it, no problem

Reply:
I suspected that was a thing. It's fine, we can get to it during some further refactor :+1: